src/hakunapi-simple-servlet-javax/src/main/java/fi/nls/hakunapi/simple/servlet/javax/operation/GetCollectionItemByIdOperation.java

```diff
@@ -138,42 +145,51 @@ public Response handle(
     }
 
     public Response getResponse(SingleFeatureWriter writer, FeatureProducer producer,
-            GetFeatureRequest request, GetFeatureCollection c) throws Exception {
-        ValueProvider feature = null;
-        try (FeatureStream features = producer.getFeatures(request, c)) {
-            if (!features.hasNext()) {
-                return ResponseUtil.exception(Status.NOT_FOUND, "Feature not found");
+            GetFeatureRequest request, GetFeatureCollection c, RequestTelemetry ftt) throws Exception {
+
+        
+        try( TelemetrySpan span = ftt.span()) {
+
+            ValueProvider feature = null;
+            try (FeatureStream features = producer.getFeatures(request, c)) {
+                if (!features.hasNext()) {
+                    return ResponseUtil.exception(Status.NOT_FOUND, "Feature not found");
+                }
+                feature = features.next();
             }
-            feature = features.next();
-        }
 
-        int srid = request.getSRID();
+            int srid = request.getSRID();
```

Comment:
Some whitespace changes here make it harder to tell what actually changed in this file. Not necessary to change, should look into moving towards some projection wide linting solution.

Reply:
Yes,
I had to reindent everything at one point due missing indentantion settings, sorry about that